metashare/api/models.py

```diff
@@ -749,9 +763,7 @@ def finalize_refresh_org(self, error=None):
 
 @receiver(user_logged_in)
 def user_logged_in_handler(sender, *, user, **kwargs):
-    from .jobs import refresh_github_repositories_for_user_job
-
-    refresh_github_repositories_for_user_job.delay(user)
```

Comment:
We were intentionally _not_ setting `user.currently_fetching_repos` here, so that users with repos do not have to wait for them to be re-fetched on every login.

Reply:
Changing this would either defeat the purpose of this PR, or would require more substantive changes to the front-end. Leaving for now, and we can see how annoying it is.

Reply:
@jgerigmeyer oh I see, so we were fetching them but not showing the spinner on the frontend. And I think our options are:
1. always fetch on login, and show the spinner (I think this will be too frustrating for me, it takes over a minute to fetch my repos)
2. always fetch on login, and make more substantive changes so the frontend can skip showing the spinner if the user already has repos
3. only fetch on login if user doesn't have any repos yet

I lean toward 3 as a quick fix (if someone has been added to a repo that MetaShare knows about, they'll need to click the resync button manually -- which is what they would have to do anyway if they were already logged in)

Reply:
Yeah, that's correct. Let me spend just a few minutes today looking into how difficult option 2 would actually be, before we jump to option 3.

Reply:
Addressed (option 2) in https://github.com/oddbird/MetaShare/commit/dce6e7e84e4f19f6084e632ce77bc1f3183a7da2

Reply:
@jgerigmeyer thanks!